gui/list.go

```diff
@@ -55,6 +57,7 @@ func UpdateMetadataLoop(m *model, delayToRefresh time.Duration) {
 func (m *model) Init() tea.Cmd {
 	m.playingItemIndex = m.list.Index()
 	m.volume = 100
+	width = tm.Width()
```

Comment:
Don't use external lib to get terminal width, you should be able to use [WindowSizeMsg.Width](https://pkg.go.dev/github.com/charmbracelet/bubbletea#WindowSizeMsg.Width) from bubbletea